pkg/daemon/update.go

```diff
@@ -197,10 +197,26 @@ func (dn *Daemon) performPostConfigChangeNodeDisruptionAction(postConfigChangeAc
 			serviceName := string(action.Restart.ServiceName)
 
 			if err := restartService(serviceName); err != nil {
-				if dn.nodeWriter != nil {
-					dn.nodeWriter.Eventf(corev1.EventTypeWarning, "FailedServiceRestart", fmt.Sprintf("Restarting %s service failed. Error: %v", serviceName, err))
+				// On RHEL nodes, this service is not available and will error out.
+				// In those cases, directly run the command instead of using the service
+				if serviceName == "coreos-update-ca-trust.service" {
```

Comment:
The workaround makes sense to me! Just a couple of suggestions:
1. maybe make these values constants (e.g. service name)
2. probably not worth the effort, but I wonder if we should add a check if we're on non-rhcos rhel before attempting the manual run? This way I guess we capture the "original failure". Although I really doubt that the service failing and the manual `update-ca-trust` failing will yield different errors, so I'm ok with having it as is as well

Reply:
1. can do!
2. yeah, I figured it wasn't worth the effort, forgiveness vs permission thing :P